metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -575,6 +582,28 @@ def __init__(self, group, stream, pos=0, sleep_on_no_data=0):
         self._token = None
 
     def _get_events(self):
+        done = RunningJob.is_done
+        status_code = RunningJob.status_code
+        if done:
+            if status_code == 1:
+                return 'Task failed with a general error. \
+                You may have a miscellaneous error(s), such as "divide by zero" and other impermissible operations.'
+            if status_code == 2:
+                return 'Misuse of shell builtins. You may have a missing keyword or command, or \
+                       permission problem (and diff return code on a failed binary file comparison).'
+            if status_code == 126:
+                return 'Command invoked cannot execute. May be a permission problem or command is not an executable.'
+            if status_code == 127:
+                return 'Command not Found. May be a possible problem with $PATH or a type.'
+            if status_code == 128:
+                return '	Invalid argument to exit.'
+            if status_code == 130:
+                return 'Script terminated by Control-C.'
+            if status_code == 255:
+                return 'Exit status out of range.'
+            else:
+                return 'Task failed.'
+
```

Comment:
In general, I don't think that BatchLogs is the best place to classify this. As you need access to the status (done and status_code), it may better be placed after _print_available in the the batch wait method.

Reply:
I was able to call the status_code using:
```python
self.info['Container'].get('exitCode').values()
```

If that's sufficient, I could leave it there, or if it isn't and you believe I should move it to where you suggest, I'll do so.